docs/esp8266/quickref.rst

```diff
@@ -235,7 +235,13 @@ and is accessed via the :ref:`machine.I2C <machine.I2C>` class::
 Real time clock (RTC)
 ---------------------
 
-See :ref:`machine.RTC <machine.RTC>` ::
+See :ref:`machine.RTC <machine.RTC>` 
+
+.. note:: Not all methods are implemented: `RTC.now()`, `RTC.irq(handler=*)` 
```

Comment:
I'd suggest to shift this block down to after the example, as the information is more "additional details" than "very important", and first-time users would want to see the examples first.

Reply:
Seems reasonable, I changed it in a new commit.